pkg/reconciler/route/route.go

```diff
@@ -176,7 +176,8 @@ func (c *Reconciler) reconcile(ctx context.Context, r *v1alpha1.Route) error {
 		Scheme: "http",
 		Host:   host,
 	}
-	r.Status.DeprecatedDomain = host
+	// TODO(mattmoor): Remove completely after 0.7 cuts.
+	r.Status.DeprecatedDomain = ""
```

Comment:
Clients should already be updated to read the new fields per M1. Why do we need to preserve this in 0.7?

Reply:
This is forcibly clearing the fields from old resources.  We can remove it after 0.7 because 0.7 will have cleared it out of any existing resources.